### PR TITLE
P1/feat/48 reactify footprint

### PR DESCRIPTION
**Pull request by elinaolsson**
Footprint is reactified and action of the month is displayed next to it. There are still some issues left to fix though but will continue locally in the same branch.

Also did some refactoring. MonthlyAction is a new component including the monthly action and also created the component MainInfo which includes FootprintContainer and MonthlyAction.

Files copied from Project 2:
* CategoryChart
* FootprintContainer (ResultContainer for Project 2)
* ResultBar
* ResultText
* YourFootprintText
* result-helper

Issues left to fix:
* Responsiveness
* Routing for button and links
* Adding a link for previous footprints

This PR is quite large, so don't hesitate to ask any questions if anything is unclear!

### CODE REVIEW

File: app/assets/main/react_components/User/MonthlyAction.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import CarouselActionItem from "./CarouselActionItem.jsx";
+
+const MonthlyAction = ({ action, user, updateLocalAccepted, categories }) => {
+  // Kolla om Monthly action finns!
```
Comment (netlight-saraandersson) [inline app/assets/main/react_components/User/MonthlyAction.jsx:5]: Remove this comment

**Review by netlight-saraandersson — APPROVED** [review]
Amazing :D

File: app/assets/main/react_components/User/ClimateActionsContainer.jsx [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
-const ClimateActionsContainer = ({ user, climateActionCategories }) => {
+const ClimateActionsContainer = ({
+  user,
+  climateActionCategories,
+  footprint,
+  commonText,
+  countryAverage,
+  modelText,
+  lang,
+  registrationsText,
```
Comment (netlight-johannadagfalk) [inline app/assets/main/react_components/User/ClimateActionsContainer.jsx:21]: Would it be reasonable to add a context to avoid this many props?

**Review by netlight-johannadagfalk — APPROVED** [review]
NICE :D check my comment about context, otherwise 👍

**Pull request merged**